tasks/linter.py

```diff
@@ -383,3 +385,29 @@ def gitlab_ci(_, test="all", custom_context=None):
         if not res["valid"]:
             print(color_message(f"Errors: {res['errors']}", "red"), file=sys.stderr)
             raise Exit(code=1)
+
+
+@task
+def releasenote(ctx):
+    """
+    Lint release notes with Reno
+    """
+    branch = os.environ.get("BRANCH_NAME")
+    pr_url = os.environ.get("PR_ID")
```

Comment:
That's a bit strange to map the env var `PR_ID` to `pr_url`.
Reading the following it looks like the `PR_ID` variables refert to `PR_URL`
```suggestion
    pr_url = os.environ.get("PR_URL")
```

Reply:
Didn't you recognise a mutation test 😄
Thanks for the check, I think it just worked because the URL is only requested to get the ID, so maybe the ID was not accessible directly on circleci